src/util/sandboxios.cpp

```diff
@@ -0,0 +1,35 @@
+#include "util/sandboxios.h"
+
+#include <QDir>
+#include <QRegularExpression>
+#include <QString>
+#include <QStringLiteral>
+#include <QtGlobal>
+
+namespace mixxx {
+
+#ifdef Q_OS_IOS
+
+static const QRegularExpression sandboxPrefixRegex = QRegularExpression(
+        QStringLiteral("^/private/var/mobile/Containers/Data/Application/"
+                       "[a-zA-Z0-9\\-]+(/.*)"));
+
+QString updateiOSSandboxPath(const QString& path) {
```

Comment:
It is hard to read when using camel case and trade mark case in one name. 
```suggestion
QString updateIosSandboxPath(const QString& path) {
```

Reply:
Personally I've found the original casing easier to parse than a differently cased version, but I'd be fine either way. If we do that, we should probably do it consistently for all methods in this PR

Reply:
I have updated the casing to `updateIOSSandboxPath` here and in the other methods, to stay consistent with the spelling we use for the iTunes importers (e.g. #12690). Does that work for you?

Reply:
That works nicely as well.